<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordWindow</class>
 <widget class="QWidget" name="PasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信息修改</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #EEF3F8, stop:1 #D7E6F5);
}

QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #C9D6E8;
    border-radius: 8px;
    padding: 8px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 2px solid rgb(145, 168, 235);
}

QLineEdit:placeholder {
    border-color: #4c65af;
    outline: none;
}

QLineEdit:focus {
    border: 2px solid #4c65af;
    outline: none;
}

QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1f49a9, stop:1 #4A8EC9);
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    border-radius: 8px;
    border: none;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #4c65af, stop:1 #3A78B5);
}

QPushButton:pressed {
    background-color: #3A78B5;
}

QLabel {
    color: #33475B;
    font-size: 16px;
    font-weight: bold;
    padding: 4px;
}

QComboBox {
    font-family: &quot;PingFang SC&quot;, &quot;Helvetica Neue&quot;, Arial, &quot;Hiragino Sans GB&quot;, &quot;Microsoft YaHei UI&quot;, &quot;Microsoft YaHei&quot;, SimSun, sans-serif;
    font-size: 16px;
    color: #333333; 
    border: 1px solid #e0e0e0;
    border-radius: 10px; 
    background-color: #ffffff; 
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
}


QComboBox:hover {
    border-color: #0078d7; 
    background-color: #f5f5f5; 
    border: none;
    outline: none;
}

QComboBox:focus {
    border-color: #0078d7;
    border: none;
    outline: none;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: #333333;
    font-size: 16px;
    padding: 8px 0;
    min-width: 100px; 
    max-height: 250px; 
    border: none;
    outline: none;
}

QComboBox QAbstractItemView::item {
    padding: 10px 15px; 
    background-color: #ffffff;
    border-radius: 8px;
    border: none;
    outline: none;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #0078d7; 
    color: #ffffff; 
    border: none;
    outline: none;
}

QComboBox QAbstractItemView::item:selected {
    background-color: #0078d7; 
    color: #ffffff; 
    border: none;
    outline: none;
}





#usernameEdit {
    font-size: 24px;
    font-weight: bold;
    color: #2B5D7C;
    text-align: center;
    background: transparent;
    border: none;  /* 移除边框 */
    outline: none; /* 移除获得焦点时的轮廓 */
}
#helloEdit{
    font-size: 24px;
    font-weight: bold;
    color: #2B5D7C;
    text-align: center;
    background: transparent;
    border: none;  /* 移除边框 */
    outline: none; /* 移除获得焦点时的轮廓 */
}
#searchButton {
    font: inherit;
    text-transform: none;
    white-space: nowrap;
    line-height: 1;
    text-align: center;
    font-size: 18px;
    color: #fff;
    text-decoration: inherit;
    background-color: #1f49a9; /* 自定义背景颜色 */
    border: none;
    border-radius: 8px;
    font-family: &quot;Microsoft YaHei&quot;, Arial, sans-serif;
}

#searchButton:hover {
    background-color:rgb(39, 86, 197); /* 悬浮状态时的背景颜色 */
}

#searchButton:focus {
    outline: none; /* 去掉焦点时的边框 */
}


   </string>
  </property>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1059</width>
     <height>700</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>700</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>700</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>720</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>171</width>
         <height>261</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>261</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="infoButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>信息修改</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="quitButton">
            <property name="text">
             <string>退出登录</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="logo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>151</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(:/images/logo.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>850</width>
        <height>700</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1080</width>
        <height>720</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>1080</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    border-top: none;         /* 不显示上边框 */
    border-left: none;        /* 不显示左边框 */
    border-right: none;       /* 不显示右边框 */
    border-bottom: 1px solid black;  /* 设置下边框为 3 像素黑色实线 */
}
</string>
         </property>
         <widget class="QPushButton" name="pushButton_pass">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>140</width>
            <height>100</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 25px &quot;微软雅黑&quot;;
color:black;
background:transparent;
</string>
          </property>
          <property name="text">
           <string>修改密码</string>
          </property>
         </widget>
         <widget class="QPushButton" name="back">
          <property name="geometry">
           <rect>
            <x>700</x>
            <y>30</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_authen">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>0</y>
            <width>140</width>
            <height>100</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 25px &quot;微软雅黑&quot;;
color:rgb(143, 143, 143);
background:transparent;
</string>
          </property>
          <property name="text">
           <string>实名认证</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>600</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QFrame" name="horizontalFrame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>300</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>470</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <widget class="QLabel" name="label">
                <property name="geometry">
                 <rect>
                  <x>80</x>
                  <y>30</y>
                  <width>81</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="text">
                 <string>原密码：</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_2">
                <property name="geometry">
                 <rect>
                  <x>80</x>
                  <y>100</y>
                  <width>81</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="text">
                 <string>新密码：</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_3">
                <property name="geometry">
                 <rect>
                  <x>70</x>
                  <y>170</y>
                  <width>81</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="text">
                 <string>确认密码：</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="oriWord">
                <property name="geometry">
                 <rect>
                  <x>170</x>
                  <y>40</y>
                  <width>241</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="placeholderText">
                 <string>请输入原密码</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="newWord">
                <property name="geometry">
                 <rect>
                  <x>170</x>
                  <y>110</y>
                  <width>241</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="toolTip">
                 <string>至少含有1个大写字母、数字、小写字母 8-16位;不能有特殊符号</string>
                </property>
                <property name="placeholderText">
                 <string>请输入新密码</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="newWordConfirm">
                <property name="geometry">
                 <rect>
                  <x>170</x>
                  <y>180</y>
                  <width>241</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="placeholderText">
                 <string>请再次输入新密码</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="horizontalFrame">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="submitButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>确认提交</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
